packages/user-service/src/helpers.ts

```diff
@@ -61,7 +61,7 @@ class UserGroupApiHelper {
           } );
           resolve( newUser.save() );
         } else {
-          resolve( {} );
+          reject();
```

Comment:
Add some helpful error. Promises shouldn't just terminate without a reason.

> Like throw in plain old JavaScript, it's customary, but not required, to reject a promise with an Error object. The benefit of Error objects is they capture a stack trace, making debugging tools more helpful.
>
> – https://web.dev/promises/#promises-arrive-in-javascript!

Reply:
actually its not an error it should send null response thats why i used resolve with null object.

Reply:
Updated